src_py/pkgdata.py

```diff
@@ -22,7 +22,24 @@ def getResource(identifier, pkgname=__name__):
 import os
 
 try:
-    from pkg_resources import resource_stream, resource_exists
+    import importlib_resources
```

Comment:
```suggestion
    import importlib.resources
```
This is a third party backport library, and I think it would be better to use the stdlib one directly (and handle the python 3.8 case in code)

Reply:
I'm not sure I understand the problem you're describing. I've provided a new solution given how I understand the problem. I have a feeling I have got this all backwards.